### PR TITLE
Add --rpc-host and --rpc-port for Validator

### PR DESCRIPTION
**Pull request by fabdarice**

### CODE REVIEW

File: website/docs/prysm-usage/parameters.md [hunk 1]
```diff
@@ -131,6 +131,8 @@ These flags are specific to launching a validator client.
 | ------------- |:-------------|
 |`--no-custom-config` | Run the beacon chain with the real parameters from phase 0.
 |`--beacon-rpc-provider` | Beacon node RPC provider endpoint. Default: localhost:4000
+|`--rpc-host` | Specify the RPC host exposed by the validator Default: localhost
```
Comment (fabdarice) [inline website/docs/prysm-usage/parameters.md:134]:
```suggestion
|`--rpc-host` | Specify the RPC host exposed by the validator. Default: localhost
```

File: website/docs/prysm-usage/parameters.md [hunk 2]
```diff
@@ -131,6 +131,8 @@ These flags are specific to launching a validator client.
...
+|`--rpc-port` | Specify the RPC port exposed by the slasher Default: 7000
```
Comment (fabdarice) [inline website/docs/prysm-usage/parameters.md:135]:
```suggestion
|`--rpc-port` | Specify the RPC port exposed by the slasher. Default: 7000
```

**Comment by rauljordan** [comment]
@fabdarice also let's add the --grpc-gateway-host and --grpc-gateway-port

**Comment by fabdarice** [comment]
@rauljordan added.

**Review by rauljordan — APPROVED** [review]

**Pull request merged**